### PR TITLE
changes for task 13 regarding creating a custom middleware and a cust…

### PR DESCRIPTION
**Pull request by Engy-Mohamed**
…om route file ,and a research about middlewares

### CODE REVIEW

File: bootstrap/app.php [hunk 1]
```diff
@@ -3,15 +3,22 @@
...
+use Illuminate\Support\Facades\Route;
 
 return Application::configure(basePath: dirname(__DIR__))
     ->withRouting(
         web: __DIR__.'/../routes/web.php',
         commands: __DIR__.'/../routes/console.php',
-        health: '/up',
+        health: '/up',                                         //using web middleware group 
+        then: function () {// (prefix )related to the uri and the (name )related to the names of the routes
+            Route::namespace('Admin')->prefix('')->name('')->middleware('web')->group(base_path('routes/admin.php'));
```
Comment (Nora-Lootfy) [inline bootstrap/app.php:15]: Cannot trick you😂

thanks ya @Engy-Mohamed

File: tasks/task13.md
Comment (Nora-Lootfy) [inline tasks/task13.md:1]: hoping yu have found it useful

**Review by Nora-Lootfy — APPROVED** [review]
Thanks ya @Engy-Mohamed 

I will be happy if you found the optional task useful because it is very important middleware group that do some crucial tasks

**Comment by Engy-Mohamed** [comment]
Yes, I enjoyed doing it

**Pull request merged**